docs/editor_manual/new_pages/index.md

```diff
@@ -1,8 +1,8 @@
 # Creating new pages
 
-![](../../_static/images/screen08_add_page_button.png)
+![](../../_static/images/screen09.1_page_more_dropdown.png)
```

Comment:
It’s now the page_more_dropdown screenshot that shows the "Add child page" button.